bugsnag-android-core/src/main/java/com/bugsnag/android/SessionTracker.java

```diff
@@ -392,7 +373,6 @@ void updateForegroundTracker(String activityName, boolean activityStarting, long
                 lastExitedForegroundMs.set(nowMs);
             }
         }
-        setChanged();
```

Comment:
What's the justification for this change?

Reply:
The call is redundant as `setChanged()` is called as a result of `notifyNdkInForeground()` being invoked - therefore this can be removed.